framework/encode/capture_manager.cpp

```diff
@@ -697,6 +697,24 @@ bool CommonCaptureManager::RuntimeTriggerDisabled()
     return result;
 }
 
+bool CommonCaptureManager::RuntimeWriteAssetsEnabled()
+{
+    CaptureSettings settings;
+    CaptureSettings::LoadRunTimeEnvVarSettings(&settings);
+    bool write_assets = settings.GetTraceSettings().runtime_write_assets;
+
+    if (previous_write_assets_ != write_assets)
```

Comment:
I struggled to understand this logic.  I expected `write_assets_` to be set from `write_assets` and for `previous_write_assets_` to be the previous value of `write_assets_`.  I don't have a good suggestion except it looks like `write_assets_` is a one-shot, so could `write_assets` be `write_assets_requested` and `previous_write_assets_` be `write_assets_already_requested_`?  (Or some such)

Reply:
I think I understand the thinking but the names didn't help explain what's going on in this block to me.  I think this is a confusing block of code with the current names but since it works and is a high-priority feature, lacking a better suggestion I think I'd like to get it in and come back to it.